app/views/payment_link.php

```diff
@@ -0,0 +1,79 @@
+<form method="post" action="payment_link">
+    <fieldset>
+        <legend>Create Payment Link</legend>
+
+        <div class="form-group row">
+            <label class="col-md-4 col-form-label">Amount</label>
+            <div class="col-md-5">
+                <input name="order[amount]" required class="form-control" placeholder="Amount"  value="29" />
+            </div>
+            <div class="col-md-3">
+                <select name="order[currency]" class="form-control">
+                    <option value="EUR">EUR</option>
+                    <option value="USD">USD</option>
+                    <option value="GBP">GBP</option>
+                </select>
+            </div>
+        </div>
+
+        <div class="form-group row">
+            <label class="col-md-4 col-form-label">Description</label>
+            <div class="col-md-8">
+                <input name="order[description]" class="form-control" placeholder="Description" />
+            </div>
+        </div>
+
+        <div class="form-group row">
+            <label class="col-md-4 col-form-label">Country</label>
+            <div class="col-md-8">
+                <select name="order[country]" class="form-control">
+                    <option value="">Dont Specify</option>
+                    <?php require_once 'partial/country.php'; ?>
+                </select>
+            </div>
+        </div>
+
+        <div class="form-group row">
+            <label class="col-md-4 col-form-label">Expiration date</label>
+            <div class="col-md-3">
+                <select name="order[exp_year]"  class="form-control">
+                    <option value="">No End</option>
+                    <?php for ($i = date('Y'); $i <= date('Y', strtotime('+10 years')); $i++) : ?>
```

Comment:
Payment links cannot exceed 90 days, at least in current configuration. +10 years is not necessary here, one year ahead should be enough